client/luigi-client.d.ts

```diff
@@ -594,14 +594,10 @@ export declare interface StorageManager {
 /**
  * Registers a listener called with the context object and the Luigi Core domain as soon as Luigi is instantiated. Defer your application bootstrap if you depend on authentication data coming from Luigi.
  * @param {Lifecycle~initListenerCallback} initFn the function that is called once Luigi is initialized, receives current context and origin as parameters
- * @param {boolean} disableTpcCheck if set to `true` third party cookie check will be disabled via LuigiClient.
  * @memberof Lifecycle
  */
-export function addInitListener(initFn: (context: Context, origin?: string) => void, disableTpcCheck?: boolean): number;
-export type addInitListener = (
-  initFn: (context: Context, origin?: string) => void,
-  disableTpcCheck?: boolean
```

Comment:
please keep the disableTcpCheck argument, it should still be possible to disable via js, e.g. in a test environment.